src/main/kotlin/lotto/LottoHandler.kt

```diff
@@ -1,11 +1,34 @@
 package lotto
 
+import lotto.view.InputView
+import lotto.view.OutputView
+
 object LottoHandler {
     fun start() {
         try {
-            val machine = buyTickets()
-            val winningTicket = processWinningNumbers()
-            val winningNumbers = processBonusNumbers(winningTicket)
+            val purchaseAmount = processPurchaseAmount()
+            val enteredTicketCount = processTicketCount()
+
+            val machine =
+                LottoMachine(
+                    purchaseAmount,
+                    enteredTicketCount,
+                )
+            machine.tickets.addAll(
+                readManualTickets(
+                    machine.manualTicketCount(),
+                ),
+            )
```

Comment:
> Don’t use any getters/setters/properties. Send message to Object.

How about ask the message to `LottoMachine`?

```suggestion
            val machine = LottoMachine(
                purchaseAmount,
                enteredTicketCount,
            )
            val manualTickets = readManualTickets(machine.manualTicketCount())
            machine.appendManualTickets(manualTickets)
```